package.json

```diff
@@ -38,8 +38,9 @@
   },
   "scripts": {
     "build": "lerna run build",
-    "check-license": "./scripts/check-license.sh",
+    "check-license": "chmod +x scripts/check-license.sh && ./scripts/check-license.sh",
```

Comment:
changing source files in a script is not a good practice. Aren't permissions already stored in Git?

If not, you could do `bash <script>`

Reply:
Didn't knew that before. Thanks.
Reverting this change.